bundle/config/resources/secret_scope.go

```diff
@@ -0,0 +1,73 @@
+package resources
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/marshal"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+)
+
+type SecretScope struct {
+	Name                   string `json:"name"`
+	InitialManagePrincipal string `json:"initial_manage_principal"`
+
+	ModifiedStatus ModifiedStatus `json:"modified_status,omitempty" bundle:"internal"`
+
+	*workspace.SecretScope
+}
+
+func (s *SecretScope) UnmarshalJSON(b []byte) error {
+	return marshal.Unmarshal(b, s)
+}
+
+func (s SecretScope) MarshalJSON() ([]byte, error) {
+	return marshal.Marshal(s)
+}
+
+func (s SecretScope) Exists(ctx context.Context, w *databricks.WorkspaceClient, name string) (bool, error) {
+	scopes, err := w.Secrets.ListScopesAll(ctx)
+	if err != nil {
+		return false, nil
+	}
```

Comment:
It seems weird that there is no way to get a scope, since its name is the identifier.

Can you double check internally and include a comment here if there really isn't one?

Reply:
I double checked - and there is not other API to get a scope by name. There are other means of checking indirectly if a given scope exists (e.g. trying to read a secret from it and then parse the errors) but they are not as semantically correct as listing the scopes.

Reply:
Thanks, can you include this as a comment as well?